src/libraries/Common/tests/System/Net/Http/ByteAtATimeContent.cs

```diff
@@ -24,6 +25,24 @@ public ByteAtATimeContent(int length, Task waitToSend, TaskCompletionSource<bool
             _millisecondDelayBetweenBytes = millisecondDelayBetweenBytes;
         }
 
+#if NETCOREAPP
```

Comment:
I'm okay with doing it this way, but I know we often use partial class + project file for conditional compilation rather than preprocessor. @davidsh, thoughts?